data/system-suspend-symbolic.svg

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
We do have a [`system-suspend`](https://github.com/elementary/icons/blob/master/actions/32/system-suspend.svg) icon; this symbolic version should be added to the icon set instead of GResourced here. And while we're at it, we should make sure to align the suspend icon metaphors.

Reply:
If we do, it isn't named correctly:
![Screenshot from 2019-07-16 16 56 46](https://user-images.githubusercontent.com/4886639/61325277-be653380-a7ea-11e9-9f6f-c39bb27671d8.png)

Reply:
See the link 🤦

https://github.com/elementary/icons/blob/master/actions/32/system-suspend.svg